test/jdk/java/lang/Thread/virtual/ThreadAPI.java

```diff
@@ -1111,12 +1109,11 @@ void testYield1() throws Exception {
      * Test Thread.yield when thread is pinned by native frame.
      */
     @Test
-    void testYield2() throws Exception {
-        assumeTrue(ThreadBuilders.supportsCustomScheduler(), "No support for custom schedulers");
+    void testYield3() throws Exception {
+        assumeTrue(VThreadScheduler.supportsCustomScheduler(), "No support for custom schedulers");
```

Comment:
Q: What was the reason to rename `testYield2()` to `testYield3()`?

Reply:
There are a lot of tests that can't be proposed to main line at this time because they depend on the object monitor work. This is one such "gap" where testYield2 can't be added.  I can re-number them if you want to avoid the gap but there are other changes that rename several test methods so I'd prefer to leave it until then.

Reply:
Okay, thanks.